boot.php

```diff
@@ -14,7 +14,7 @@
 }
 
 // bessere SEO urls für den image-manager
-if (!rex::isBackend()) {
```

Comment:
Das ist bereits enthalten in der package.yml. Was passiert jedoch, wenn yrewrite deaktiviert wird? Wird das abgefangen? Wenn ja, muss das auch nicht rein.